readme.md

```diff
@@ -28,14 +28,113 @@ This is the draft of some future version of the CCS specification.
+      3. Access  
+         1) Updated examples (already backpatched to 2023-06)  
+      4. Contests  
+         1) Clarify wording for **countdown\_pause\_time** and description below  
+         2) Change type of penalty\_time to **RELTIME**  
+         3) Change examples to use **RELTIME** for **penalty\_time**  
+      5. Judgement Types  
+         1) Added description of new property **simplified\_judgement\_type**  
+         2) Added example for **simplified\_judgement\_type**  
+      6. Languages  
+         1) Change gcc example to include **\-o a.out**  
+         2) **ERROR** in example for **“id”: “cpp”** \-  can not use gcc to build c++ programs  \- should be fixed  
+      7. Problems  
+         1) Added descriptions for new properties **memory\_limit**, **output\_limit**, **code\_limit, attachments**  
+         2) Clarify maximum score for problems in **score** type contests.  
+         3) Clarify that the **rgb** property should not include an alpha channel.  
+         4) Fixed examples to include newly added properties  
+         5) Fixed JSON formatting error in example.  
+      8. Organizations  
+         1) Added descriptions for new country subdivision properties  
+         2) Updated examples to include country subdivisions  
+      9. Teams  
+         1) Update description of **id** property to no longer include suggestion of using the **id** as the seat number since **label** does that.  
+         2) Remove specific mention of no implied order ID’s in an array for **team\_ids** and **group\_ids** since it’s covered in the *General Design Principles*.  
+         3) Allow HLS video (application/vnd.apple.mpegurl) for **video**, **desktop, webcam** and **reaction** properties.  
+      10. State  
+          1) Minor wording changes that affect nothing  
+      11. Submissions  
+          1) Added text indicating why the **id** should be a small number  
+          2) Add description of new property **account\_id**.  
+          3) Clarify what a missing **team\_id** property implies.  
+          4) Spelling fixes (entrypoint \-\> entry point)  
+          5) Clarify that the **data** property for the **files** array is a *base64 encoded zip* file  
+      12. Judgements  
+          1) Add description of new properties: **current, simplified\_judgement\_type**  
+      13. Clarifications  
+          1) Change **team\_clar** capability to **post\_clar** capability  
+          2) Added description of how the new **to\_team\_ids** and **to\_group\_ids** properties should be used.  
+          3) Provide more detail on broadcast clarification responses  
+          4) Consistently use “judges” instead of “jury” in descriptions.  
+          5) Update rules for **post\_clar** capability (which fields may be specified by what roles, etc).  
+      14. Awards  
+          1) Added descriptions of new “Bill” rules awards for honors, high-honors, highest-honors  
```

Comment:
Do we want to name these "Bill" rules? Maybe better "ICPC World Finals specific".